docs/source/certs_management.md

```diff
@@ -0,0 +1,381 @@
+
+
+#### Orderer TLS Certificate
+
+**Description**: TLS certificate for the ordering node.
+
+**Location**: Dependent on implementation:
+
+<pre>
+ordererca/
+└── orderer1
+    ├── msp
+    └── tls
+        ├── cacerts
+        ├── keystore
+        |   └── key.pem
+        ├── signcerts
+        │   └── cert.pem
+        ├── tlscacerts
+        │   └── tls-localhost-7053.pem
+        └── user
+  </pre>
+
+**Impact if expired**: Production outage. Ordering nodes are no longer allowed to participate in cluster.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-orderer)
+
+
+### Admin Certificates
+
+Ordering Service Organization Channel Admin certificates and Peer Service Organization Channel Admin certificates are issued for each organization.
+
+#### Ordering Service Organization Channel Admin Certificate
+
+**Description**: Admin certificate used to administer the ordering service and submit or approve channel updates. The associated admin identity is enrolled with the Organization CA before the node and Organization MSP are created.
+
+**Location**: Dependent on implementation:
+
+<pre>
+ordererca/
+└── ordereradmin
+└── msp
+    ├── admincerts
+    │   └── cert.pem
+    ├── cacerts
+    │   └── localhost-7053.pem
+    ├── keystore
+    │   └── key.pem
+    ├── signcerts
+    │   └── cert.pem
+    └── user
+</pre>
+
+**Impact if expired**: Transactions can continue to work successfully. Cannot modify channels from a client application or manage the orderer from the console.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org0-s-admin)
+
+
+#### Peer Service Organization Channel Admin Certificate
+
+**Description** - Admin certificate used to administer the peer and install smart contracts on the peer. Can also be configured to manage the application channel that the peer belongs to. The associated admin user is registered with the Organization CA before the peer and Organization MSP are created.
+
+**Location** - Dependent on implementation:
+
+<pre>
+org1ca/
+└── org1admin
+└── msp
+├── admincerts
+│     └── cert.pem
+├── cacerts
+│     └── localhost-7053.pem
+├── keystore
+│     └── key.pem
+├── signcerts
+│     └── cert.pem
+└── user
+</pre>
+
+**Impact if expired**: Transactions can continue to work successfully. Cannot install new smart contracts from a client application or manage the peer from the console.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org1-s-admin)
+
+
+
+### Client Certificates
+
+**Description**: Three types of Client certificates are issued for each organization:
+
+1. Certificates used to authenticate an identity to the Gateway when submitting transactions
+2. Client-side TLS certificates used to establish a secure network link to a Gateway peer
+3. Certificate for a client SDK user if no gateway is accessed.
```

Comment:
There are two types of certificates issued to a client (same story as peers and orderers and we can use consistent language and structure).
1 - Organization enrollment certificate - authenticates the client identity when communicating with peers and orderers (this covers your 1 and 3).
2 - TLS certificate - authenticates client communications, only required if mutual TLS is configured.

Reply:
ok.